backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1666,6 +1675,7 @@ class ConfigSchema(marshmallow.Schema):
     email_notification_activated = marshmallow.fields.Bool()
     new_user_invitation_do_notify = marshmallow.fields.Bool()
     webdav_enabled = marshmallow.fields.Bool()
+    translation_service__enabled = marshmallow.fields.Bool()
```

Comment:
We already have _activated and _enabled, and now we will have __enabled. I know I should have reacted before, but are we sure we want to introduce a new inconsistency here?

Reply:
i don't have strong opinion here. Each solution has pro and cons.